packages/plugin-core/src/services/EngineAPIService.ts

```diff
@@ -83,62 +84,71 @@ export class EngineAPIService implements DEngineClient {
   get trustedWorkspace(): boolean {
     return this._trustedWorkspace;
   }
+
   set trustedWorkspace(value: boolean) {
     this._trustedWorkspace = value;
   }
 
-  public get notes(): NotePropsDict {
```

Comment:
nit: Why the removal of `public`? Was this done by an auto-code formatter? 

I understand that `public` is the default access level in typescript - but I personally like it there to explicitly indicate that it's meant for external consumption.

Reply:
It was an automatic refactor. 

Will add it back.